src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -723,15 +825,21 @@ public Task<User> GetMeAsync(CancellationToken cancellationToken = default)
             int replyToMessageId = default,
             IReplyMarkup replyMarkup = default,
             CancellationToken cancellationToken = default,
-            string foursquareType = default
+            string foursquareType = default,
+            string googlePlaceId = default,
+            string googlePlaceType = default,
+            bool allowSendingWithoutReply = default
```

Comment:
The default convention in .NET world is to setup `CancellationToken` at the end of the calling params.